openwisp_users/models.py

```diff
@@ -1,5 +1,6 @@
 import swapper
 from django.contrib.auth.models import Group as AbstractGroup
+from django.db import IntegrityError
```

Comment:
Please make sure to run `openwisp-qa-format` and then `./run-qa-checks` to make sure the QA checks are passing.